components/pages/privacy/resourceAccess/show/index.jsx

```diff
@@ -216,199 +219,181 @@ export default function AgentResourceAccessShowPage({ type }) {
   if (shouldUpdate) {
     return <Spinner />;
   }
-  // FIXME: using a conditional dataset provider wrapper while we mock app profiles
-  const ConditionalWrapper = ({ condition, wrapper, children }) =>
-    condition ? wrapper(children) : children;
 
   const renderAvatar = () => {
-    if (type === schema.SoftwareApplication) {
-      return <AppAvatar profileIri={decodedIri} />;
+    if (agentTypes.includes(schema.SoftwareApplication)) {
```

Comment:
is there anyway we can export this function somewhere global and use it whenever we render an avatar (and then get rid of the other avatar components?)

Reply:
Let's keep this as in for now. As mentioned in the PR, description, we have different "versions" of Avatars and need to do more work on it but I would make that a separate PR: 

> Consolidating the multiple ways we display profiles: just Avatars when no profile available (fallback and WebID), Avatar and Profile in Privacy page with Tabs, entire Profile in Contacts, entire editable Profile in Profile page... and so on.